src/v2/Apps/Auction/Components/Form/initialValues.ts

```diff
@@ -1,5 +1,15 @@
 import { Address } from "v2/Components/AddressForm"
 
+export interface FormValuesForRegistration {
+  address?: Address
+  creditCard?: string
+  agreeToTerms: boolean
+}
+
+export interface FormValuesForBidding extends FormValuesForRegistration {
```

Comment:
```suggestion
export interface FormValuesForBiddingWhileRegistering extends FormValuesForRegistration {
```

just because there is also a "just bid" workflow, so I think a more specific name might be useful

Reply:
There are already [three different validation schemas](https://github.com/artsy/force/pull/5734/files?diff=unified&w=1#diff-01b6b6bd8c1af9ce347c18511c76823dR36-R53) for ensuring the distinction among the possible cases and I'm not sure if we need to conditionally fork the logic even for types.